lib/job.jsonnet

```diff
@@ -301,7 +301,7 @@ function(jobName, agentEnv={}, stepEnvFile='', patchFunc=identity) patchFunc({
     annotations: annotations,
   },
   spec: {
-    backoffLimit: 0,
+    backoffLimit: std.parseInt(env.BUILDKITE_PLUGIN_K8S_JOB_BACKOFF_LIMIT),
```

Comment:
we need to set a default value otherwise this would likely fail with an undefined variable https://github.com/EmbarkStudios/k8s-buildkite-plugin/blob/713aa2964070d168b89f508245205534b0ce22ea/lib/job.jsonnet#L80

Reply:
ah ok, i thought having that in the plugin configuration was enough, done